src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1388,7 +1388,8 @@ void PolicyHandler::OnSnapshotCreated(const BinaryMessage& pt_string) {
   }
 
   SendMessageToSDK(pt_string, urls.at(current_app).url.at(current_url));
-  current_url++;
+  if (!(urls.at(current_app).app_id == policy::kDefaultId))
```

Comment:
@indlu 
What if 'default' will have more than one URL? I suppose only first of default will be used and other will be skipped.